plugins/importexport/crossref/templates/index.tpl

```diff
@@ -0,0 +1,113 @@
+{**
+ * plugins/importexport/crossref/templates/index.tpl
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * List of operations this plugin can perform
+ *}
+{strip}
+{assign var="pageTitle" value="plugins.importexport.crossref.displayName"}
+{include file="common/header.tpl"}
```

Comment:
FYI, this whole stanza can be made a little cleaner/briefer:

    {include file="common/header.tpl" pageTitle="plugins.importexport.crossref.displayName"}